doc/sphinx/azhelpgen/azhelpgen.py

```diff
@@ -62,7 +32,8 @@ def make_rst(self):
                invocation_cls=AzCliCommandInvoker,
                parser_cls=AzCliCommandParser,
                help_cls=AzCliHelp)
-        help_files = get_help_files(az_cli)
```

Comment:
I would like to see this temporarily reverted so we can verify that your script would have in fact caught the discrepancy.

Reply:
Will do!